tools/tablemerge/.shed.yml

```
@@ -0,0 +1,9 @@
+categories: [Metabolomics]
```

I think this might have to be changed to either be a proper list
```suggestion
categories:
  - Metabolomics
```